src/components/MakeWidget/MakeWidget.tsx

```diff
@@ -204,8 +204,8 @@ const MakeWidget: FC = () => {
         ? wrappedTokenAddresses[chainId!]
         : takerTokenAddress;
 
-    const formattedMakerAmount = stringToSignificantDecimals(makerAmount, 4);
-    const formattedTakerAmount = stringToSignificantDecimals(takerAmount, 4);
+    const formattedMakerAmount = stringToSignificantDecimals(makerAmount, 6);
+    const formattedTakerAmount = stringToSignificantDecimals(takerAmount, 6);
```

Comment:
like you said, dust was lowered for me when createOrder takes in a non rounded makerAmount. could move stringToSignificantDecimals out of the setState.

Reply:
Yes I tried that but then the `toAtomicString` could fail give an `fractional component exceeds decimals` errors. This is because the smart contract's `signerAmount` has to be an integer.

Reply:
Ah I understand now. Oops!